From 6af4103f8f97fe69b664b071c58d0f91c0f45be8 Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <thomas.preudhomme@celest.fr>
Date: Mon, 11 Jul 2011 16:06:07 +0200
Subject: Fix array_test: test take no parameter

array_test is called without any parameter although it is defined with
one parameter. Fix this by declaring 'a' as a local variable.

Origin: vendor
Forwarded: http://repo.or.cz/w/tinycc.git/commit/4ccb5662cbfbf3e8c58a1b642fd3446fcce55e16
Last-Updated: 2011-07-12
Applied-Upstream: commit: 4ccb5662cbfbf3e8c58a1b642fd3446fcce55e16
---
 tests/tcctest.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/tcctest.c b/tests/tcctest.c
index 8d558e3..1ca2f6b 100644
--- a/tests/tcctest.c
+++ b/tests/tcctest.c
@@ -610,9 +610,9 @@ void scope_test()
     printf("g5=%d\n", g);
 }
 
-void array_test(int a[4])
+void array_test()
 {
-    int i, j;
+    int i, j, a[4];
 
     printf("array:\n");
     printf("sizeof(a) = %d\n", sizeof(a));
